resources/js/components/MarkdownPreview.vue

```diff
@@ -1,14 +1,31 @@
 <template>
-	<div ref="preview" class="markdown-body"
+	<div class="markdown-body"
 		v-html="compiledHTML"
 	></div>
 </template>
 <script>
 import md from "markdown-it";
 import mk from "markdown-it-katex";
-import hljs from 'highlight.js';
-import mdhljs from "markdown-it-highlightjs";
-import debounce from 'lodash/debounce';
+import mdhljs from "markdown-it-highlightjs/core";
+import hljs from 'highlight.js/lib/core';
```

Comment:
Reduce the bundle size, pick only the languages needed.

Reply:
Was this done?

Reply:
It doesn't look like it though: https://github.com/csdu/sankalan-portal/blob/master/resources/js/components/MarkdownPreview.vue#L10-L11.

Reply:
Ah! This PR never got merged into the master. Good catch! wondering how you found this! 🤔

I was a stacked PR, when I merged the base PR, didn't change the base branch for this to master. So it was actually merged to `laravel-v12` branch. I'll reopen another PR.

Reply:
> Ah! This PR never got merged into the master. Good catch! wondering how you found this! 🤔

I just checked and verified, manually.